### PR TITLE
wip: set up CI

### PR DESCRIPTION
**Pull request by TOTBWF**
Will this work on the first try???

### CODE REVIEW

**Comment by favonia** [comment]
For `_opam`, please use the updated .gitignore from other repos.

File: .github/workflows/ocaml.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  run:
+    strategy:
+      matrix:
+        include:
+          - ocaml-compiler: "ocaml-base-compiler.5.0.0~alpha0"
```
Comment (favonia) [inline .github/workflows/ocaml.yaml:12]: alpha1 is already out

**Comment by favonia** [comment]
I think this is a case where we should really squash all commits into one.

**Comment by TOTBWF** [comment]
Yeah, no sense in preserving this history. This is not a place of honor, no highly esteemed deed is commemorated here.

**Comment by favonia** [comment]
@TOTBWF Wait, the `.gitignore` file was not corrected...

**Pull request merged**